sbi/simulators/simutils.py

```diff
@@ -134,6 +135,87 @@ def simulation_wrapper(
     return parameters, observations
 
 
+def simulation_wrapper_batch(
+    simulator: Callable,
+    parameter_sample_fn: Callable,
+    num_samples: int,
+    simulation_batch_size: int,
+    x_dim: torch.Size,
+):
```

Comment:
Return should be annotated in the signature as well, just add ` -> (torch.Tensor, torch.Tensor)` here